packages/medusa/src/api/routes/admin/notes/get-note.ts

```diff
@@ -1,8 +1,11 @@
+import NoteService from "../../../../services/note"
+
 /**
  * @oas [get] /notes/{id}
  * operationId: "GetNoteNote"
  * summary: "Get Note"
```

Comment:
Not your responsibility but, this should be `GetNotesNote` would you mind including this in this PR?